### PR TITLE
[VEN-1132] Market table now has sorting options when in mobile view (master)

### PR DESCRIPTION
**Pull request by gleiser-oliveira**
## Jira ticket(s)

VEN-1132

## Changes

- Mobile viewport now lets the user is sort markets by: total supply, supply APY, liquidity, total borrow and borrow APY.

### CODE REVIEW

File: src/components/Table/TableCards.tsx [hunk 1]
```diff
@@ -1,72 +1,133 @@
...
+          label: column.label,
+        };
+
+        return [...acc, option];
+      }, [] as SelectOption[]),
+    [columns],
+  );
+
+  console.log(selectOptions);
+  console.log(order);
```
Comment (therealemjy) [inline src/components/Table/TableCards.tsx]: An oversight from coding hehe

**Review by therealemjy — CHANGES_REQUESTED** [review]
It works great on mobile but you'll need to hide the select on higher resolutions:
Screen Shot 2023-03-08 at 11 28 18

File: src/components/Table/TableCards.tsx (hunk 1 above)
Comment (gleiser-oliveira) [reply src/components/Table/TableCards.tsx]: 🙈

**Comment by gleiser-oliveira** [comment]
> It works great on mobile but you'll need to hide the select on higher resolutions

Indeed, thanks for catching that! I was under the false impression it was hidden, but it was there all the way down the list. Fixed.

**Review by therealemjy — APPROVED** [review]

**Pull request merged**